include/random_walks/ellipsoid_walks/math_functions.h

```diff
@@ -0,0 +1,45 @@
+// Code from https://github.com/rzrsk/vaidya-walk
+
+#ifndef PWALK_UTIL_MATH_FUNCTIONS_HPP_
+#define PWALK_UTIL_MATH_FUNCTIONS_HPP_
+
+#include <Eigen/Dense>
+#include <boost/random.hpp>
+#include <cmath>
+
+//namespace pwalk {
```

Comment:
if not used remove it

Reply:
Thanks. I have removed all unused comments